raiden-ts/src/messages/utils.ts

```diff
@@ -185,6 +194,28 @@ export function packMessage(message: Message) {
           encode(message.secret, 32),
         ]),
       ) as HexString<44>;
+    case MessageType.TO_DEVICE:
+      return hexlify(
+        concat([
+          encode(CMDIDs[message.type], 1),
+          encode(0, 3),
+          encode(message.message_identifier, 8),
+        ]),
+      );
+    case MessageType.WITHDRAW_REQUEST:
+    case MessageType.WITHDRAW_EXPIRED:
+    case MessageType.WITHDRAW_CONFIRMATION:
+      return hexlify(
+        concat([
+          encode(message.token_network_address, 20),
+          encode(message.chain_id, 32),
+          encode(MessageTypeId.WITHDRAW, 32),
+          encode(message.channel_identifier, 32),
+          encode(message.participant, 20),
+          encode(message.total_withdraw, 32),
+          encode(message.expiration, 32),
+        ]),
+      );
```

Comment:
Just for completeness, could you add the size of the packed withdraw message?
```suggestion
      ) as HexString<200>;
```